osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -62,6 +62,11 @@ private void load(OsuConfigManager config, SkinManager skins)
                     LabelText = "Beatmap hitsounds",
                     Bindable = config.GetBindable<bool>(OsuSetting.BeatmapHitsounds)
                 },
+                new SettingsCheckbox
+                {
+                    LabelText = "Color the sliderballs according to the color of the sliders",
```

Comment:
I'd follow stable text:
```suggestion
                    LabelText = "Use combo colour as tint for slider ball",
```